DuckDuckGo/MainWindow/MainViewController.swift

```diff
@@ -34,6 +35,7 @@ final class MainViewController: NSViewController {
     let findInPageViewController: FindInPageViewController
     let fireViewController: FireViewController
     let bookmarksBarViewController: BookmarksBarViewController
+    let featureFlagger: FeatureFlagger
```

Comment:
Is it actually used?

Reply:
Ah no, I mentioned it to you yesterday but forgot to remove it :) sorry and thanks for the reminder!

Reply:
Apologies. It actually is required because it's used in MainMenuActions.swift